includes/api/class-urlslab-api-cache-rules.php

```diff
@@ -41,6 +41,21 @@ public function register_routes() {
 				),
 			)
 		);
+		register_rest_route(
+			self::NAMESPACE,
+			$base . '/write_htaccess',
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
```

Comment:
its better to use POST or PUT request here. the endpoint has sideeffects. GET request is usually just for fetching data

Reply:
GET is also because access through admin header menu of other api calls of same type.